src/main.ts

```diff
@@ -131,18 +131,21 @@ function loadTscConfig(args: string[]):
 export function toClosureJS(
     options: ts.CompilerOptions, fileNames: string[], settings: Settings,
     writeFile?: ts.WriteFileCallback): tsickle.EmitResult {
-  const compilerHost = ts.createCompilerHost(options);
-  const program = ts.createProgram(fileNames, options, compilerHost);
   // Use absolute paths to determine what files to process since files may be imported using
   // relative or absolute paths
-  const filesToProcess = new Set(fileNames.map(i => path.resolve(i)));
+  const absoluteFileNames = fileNames.map(i => path.resolve(i));
+
+  const compilerHost = ts.createCompilerHost(options);
+  const program = ts.createProgram(fileNames, options, compilerHost);
+  const filesToProcess = new Set(absoluteFileNames);
   const transformerHost: tsickle.TsickleHost = {
     shouldSkipTsickleProcessing: (fileName: string) => {
       return !filesToProcess.has(path.resolve(fileName));
     },
     shouldIgnoreWarningsForPath: (fileName: string) => false,
     pathToModuleName: cliSupport.pathToModuleName,
     fileNameToModuleId: (fileName) => fileName,
+    rootModulePath: getCommonParentDirectory(absoluteFileNames),
```

Comment:
can you use `options.rootDir || getCommon...`? `rootDir` is the root of the source paths to generate, so it matches this well and should be preferred if the user sets it.